zenodo/modules/records/data/objecttypes.json

```diff
@@ -238,13 +238,13 @@
   "internal_id": "publication-section",
   "id": "https://zenodo.org/objecttypes/publication/section#",
   "title": {
-    "en": "Book section"
+    "en": "Book chapter"
   },
   "title_plural": {
-    "en": "Book sections"
+    "en": "Book chapters"
   },
   "schema.org": "https://schema.org/ScholarlyArticle",
-  "datacite": {"general": "Text", "type": "Book section"},
+  "datacite": {"general": "Text", "type": "Book chapter"},
```

Comment:
I would say we keep these as they were since they're not related to the DataCite export (but probably some kind of human-friendly text on the UI).